CMake/UsingCMakewithAMDROCm.rst

```diff
@@ -132,8 +132,30 @@ ROCm CMake Packages
 +===========+==========+=======================================================+
 | HIP       | hip      | hip::host, hip::device                                |
 +-----------+----------+-------------------------------------------------------+
+| rocPRIM   | rocprim  | roc::rocprim                                          |
++-----------+----------+-------------------------------------------------------+
+| rocThrust   | rocthrust  | roc::rocblas                                      |
++-----------+----------+-------------------------------------------------------+
+| hipCUB   | hipcub  | roc::rocblas                                            |
++-----------+----------+-------------------------------------------------------+
+| rocRAND   | rocrand  | roc::rocrand                                          |
++-----------+----------+-------------------------------------------------------+
 | rocBLAS   | rocblas  | roc::rocblas                                          |
 +-----------+----------+-------------------------------------------------------+
+| rocSOLVER   | rocsolver  | roc::rocsolver                                    |
++-----------+----------+-------------------------------------------------------+
+| hipBLAS   | hipblas  |                                       |
++-----------+----------+-------------------------------------------------------+
+| rocFFT   | rocfft | roc::rocfft                                              |
++-----------+----------+-------------------------------------------------------+
+| hipFFT   | hipfft  |                                                 |
++-----------+----------+-------------------------------------------------------+
+| rocSPARSE   | rocsparse  | roc::rocsparse                                    |
++-----------+----------+-------------------------------------------------------+
+| hipSPARSE   | hipsparse  | roc::rocblas                                          |
```

Comment:
This looks like a type as well.

Reply:
Sorry to waste your time. I want to get the component owners to review this. But I cannot add them as reviewers. Need to get them into this organization.